lambdas/data-migration1/tests/test-async-operations.js

```diff
@@ -133,16 +133,56 @@ test.serial('migrateAsyncOperationRecord migrates asyncOperation record with und
   );
 });
 
-test.serial('migrateAsyncOperationRecord throws RecordAlreadyMigrated error for already migrated record', async (t) => {
-  const fakeAsyncOp = generateFakeAsyncOperation();
+test.serial('migrateAsyncOperationRecord throws RecordAlreadyMigrated error if already migrated record is newer', async (t) => {
+  const fakeAsyncOp = generateFakeAsyncOperation({
+    updatedAt: Date.now(),
+  });
 
   await migrateAsyncOperationRecord(fakeAsyncOp, t.context.knex);
+
+  const olderFakeAsyncOp = {
+    ...fakeAsyncOp,
+    updatedAt: Date.now() - 1000, // older than fakeAsyncOp
+  };
+
   await t.throwsAsync(
-    migrateAsyncOperationRecord(fakeAsyncOp, t.context.knex),
+    migrateAsyncOperationRecord(olderFakeAsyncOp, t.context.knex),
     { instanceOf: RecordAlreadyMigrated }
   );
 });
 
+test.serial('migrateAsyncOperationRecord updates an already migrated record if the updated date is newer', async (t) => {
+  const fakeAsyncOp = generateFakeAsyncOperation({
+    updatedAt: Date.now() - 1000,
+  });
+  await migrateAsyncOperationRecord(fakeAsyncOp, t.context.knex);
+
+  const newerFakeAsyncOp = generateFakeAsyncOperation({
+    ...fakeAsyncOp,
+    updatedAt: Date.now(),
+  });
+  await migrateAsyncOperationRecord(newerFakeAsyncOp, t.context.knex);
+
+  const createdRecord = await t.context.knex.queryBuilder()
```

Comment:
Can we update this to use the model `get()`? I'd like to standardize on that instead of having the table name strings repeated throughout our code

Reply:
https://github.com/nasa/cumulus/pull/2162/commits/7027bf1243c36dac3abc1052c1e7b28bdd60a305